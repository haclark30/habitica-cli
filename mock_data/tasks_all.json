{
    "data": [
        {
            "type": "habit",
            "text": "test habit 1",
            "up": true,
            "down": false,
            "counterUp": 10
        },
        {
            "type": "daily",
            "text": "test daily 1",
            "completed": true,
            "streak": 5,
            "isDue": true
        }
    ]
}