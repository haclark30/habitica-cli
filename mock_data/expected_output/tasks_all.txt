==DAILIES==
[x] test daily 1 (5 days in a row)
==HABITS==
test habit 1
 --10 â†‘
